projects/template.md

```diff
@@ -132,6 +132,28 @@ submitted.
     * If selected, please attach a paragraph detailing the information which will be presented and any steps which will be taken to prevent this from potentially misleading the public.
   2. [ ] Project results will not present information which might lead to PKT price speculation
 
+## Use of Resources
+Each milestone report will be accompanied by:
+
+* [ ] Itemization of project expenditures, including description, date and price (in national currency)
+  * [ ] Justification of this itemization in an audit certificate
+* [ ] Dates when PKT was converted to national currency
```

Comment:
I think this is super important. Otherwise the NS could be at risk of holding on to PKT until it's worth more, even if it strings along the developers.